modules/data.remote/R/Prep_AGB_IC_from_geotiff.R

```diff
@@ -0,0 +1,43 @@
+#' Extract ensemble above ground biomass density from pre-existing GeoTIFF files for the SDA workflow.
+#' Note that, this function only works for those products who have both mean and uncertainty GeoTIFF images prepared.
+#' For example, products like: DOI: https://doi.org/10.3334/ORNLDAAC/1763.
+#'
+#' @param site_info Bety list of site info including site_id, lon, and lat.
+#' @param paths.list list containing file paths for `mean` and `uncertainty` datasets.
+#' @param ens ensemble number.
+#'
+#' @return A data frame containing sampled above ground biomass densities, each column represent each site.
+#' @export
+#' 
+#' @examples
+#' @author Dongchen Zhang
+#' @importFrom magrittr %>%
+Prep_AGB_IC_from_geotiff <- function(site_info, paths.list, ens) {
```

Comment:
This function appears to make a lot of product-specific assumptions and isn't general to *ANY* geotiff containing AGB data. At this point I'd recommend just changing the name to something that's data product specific, and then when you encounter other geotiffs with differently formatted AGB data you can think more explictly about how to generalize this function (e.g. reading in metadata on variable names, projection, etc) versus writing different functions for different products.

Reply:
Fixed.